examples/kuka_iiwa_arm/kuka_simulation.cc

```diff
@@ -0,0 +1,228 @@
+/// @file
+///
+/// Implements a simulation of the KUKA iiwa arm.  Like the driver for the
+/// physical arm, this simulation communicates over LCM using lcmt_iiwa_status
+/// and lcmt_iiwa_command messages. It is intended to be a be a direct
+/// replacement for the KUKA iiwa driver and the actual robot hardware.
+
+#include <list>
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "drake/common/find_resource.h"
+#include "drake/geometry/dev/scene_graph.h"
+#include "drake/geometry/geometry_visualization.h"
+#include "drake/manipulation/kuka_iiwa/iiwa_command_receiver.h"
+#include "drake/manipulation/kuka_iiwa/iiwa_status_sender.h"
+#include "drake/math/rigid_transform.h"
+#include "drake/multibody/parsing/parser.h"
+#include "drake/multibody/tree/revolute_joint.h"
+#include "drake/multibody/tree/multibody_tree.h"
+#include "drake/systems/framework/diagram_builder.h"
+#include "drake/systems/primitives/demultiplexer.h"
+#include "drake/systems/primitives/discrete_derivative.h"
+#include "drake/systems/analysis/simulator.h"
+#include "drake/systems/lcm/lcm_interface_system.h"
+#include "drake/systems/lcm/lcm_publisher_system.h"
+#include "drake/systems/lcm/lcm_subscriber_system.h"
+#include "drake/multibody/plant/multibody_plant.h"
+#include "drake/lcmt_iiwa_command.hpp"
+#include "drake/lcmt_iiwa_status.hpp"
+#include "examples/kuka_iiwa_arm/kuka_torque_controller.h"
+
+namespace dairlib {
+namespace examples {
+namespace kuka_iiwa_arm {
+
+ using Eigen::Vector3d;
+ using Eigen::VectorXd;
+
+ using drake::geometry::SceneGraph;
+ using drake::manipulation::kuka_iiwa::IiwaCommandReceiver;
+ using drake::manipulation::kuka_iiwa::IiwaStatusSender;
+ using drake::math::RigidTransform;
+ using drake::math::RollPitchYaw;
+ using drake::multibody::Joint;
+ using drake::multibody::MultibodyPlant;
+ using drake::multibody::RevoluteJoint;
+ using drake::multibody::SpatialInertia;
+ using drake::multibody::ModelInstanceIndex;
+ using drake::systems::StateInterpolatorWithDiscreteDerivative;
+
+int DoMain() {
+
+  std::unique_ptr<drake::multibody::MultibodyPlant<double>> owned_world_plant =
```

Comment:
Since you're already `using drake::multibody::MultibodyPlant`, this can all just be 
`MultibodyPlant` instead of `drake::multibody::MultibodyPlant` (and similar instances)